sycl/test-e2e/Matrix/SG32/element_wise_abc.cpp

```diff
@@ -11,7 +11,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-#include <cstddef>
```

Comment:
cstddef is not included in common.hpp and it is needed for size_t. I think we should keep it.

Reply:
The tests pass. `size_t` is also included indirectly through most STL libraries. In this case it is included through ` #include <iostream>`, which is imported through `common.hpp`. See https://godbolt.org/z/Gh1GTefeW for an example.

Reply:
ok, then please, remove it from `sycl/test-e2e/Matrix/element_wise_abc.cpp` as well.